### ISSUE TITLE
grad_req in multi-task example 

### ISSUE THREAD

**Issue by slyforce** [apache/incubator-mxnet#10397]
I'm referring to the python script in example/multi-task/example_multi_task.py 

As far as I understood gradients need to be accumulated between two loss functions, since you are performing a backward step on two different components onto a single symbol (in the example, f3 gets gradients from sm1 and sm2). However, the default for this example is that grad_req = 'write', implying that one softmax backward step gets overwritten by the next. 

Why is 'write' sufficient in this case? In which cases of gradient accumulation should 'add' be used?

**Comment by Roshrini** [comment]
Please use the user forum for how-to questions. You'll get quicker responses there. https://discuss.mxnet.io/
@reminisce Can you please add labels-
Question, python
